MainRepository/Program.cs

```diff
@@ -1,7 +1,25 @@
-﻿internal class Program
+﻿// Program to find the digital root of given number.
+for (; ; )
 {
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+   Console.Write ("Enter the Number or press [e] to exit: ");

```

Comment:
input, n, sum and temp can be declared outside for loop as each time new variables are initialized.